ocs_ci/ocs/resources/mcg.py

```diff
@@ -582,39 +582,56 @@ def _check_mirroring():
             )
             assert False
 
-    def check_backingstore_state(self, backingstore_name, desired_state):
+    def check_backingstore_state(
+        self,
+        backingstore_name,
+        desired_state,
+        timeout=180
```

Comment:
Great change!
I recommend bringing the default timeout up to 600 seconds; Sometimes, the change takes a while to kick into action on the AWS side, and thus on the NooBaa side too

Reply:
Thank you, updated.